WMF Framework/Remote Notifications/RemoteNotificationsAPIController.swift

```diff
@@ -218,3 +258,11 @@ extension RemoteNotificationsAPIController {
         return session.hasValidCentralAuthCookies(for: .mediawiki)
     }
 }
+
+private extension Array {
```

Comment:
could this be a public extension? would be good to use elsewhere

Reply:
sure 👍